packaging/nuget/package.ps1

```diff
@@ -31,4 +31,4 @@ foreach ($ProjectName in $ProjectsToPack) {
     }
 }
 
-Get-ChildItem $IntermediatePackagesDir\$Configuration -Filter *.nupkg | ? {$_.Name -NotLike "*.symbols.nupkg"} | Copy-Item -Destination $PackagesDir
+Get-ChildItem $IntermediatePackagesDir -Filter *.nupkg | ? {$_.Name -NotLike "*.symbols.nupkg"} | Copy-Item -Destination $PackagesDir
```

Comment:
Are we not appending the configuration to the output anymore? Why? So, if I build with debug and then build with release it will get stomped? Or worse, incremental compilation will think the output is already there and won't build again.